### DISCUSSION TITLE
Error when loading converted waymo dataset.

### DISCUSSION THREAD

**Discussion by JiSuanJiDaWang** [open-mmlab/mmdetection3d#2056] [Q&A]
I train the model on waymo and encountered the following error:
NotImplementedError: Currently we only support load image from kitti andnuscenes datasets.
I thought version1.1 supports waymo mono training.
![image](https://user-images.githubusercontent.com/82489384/203930814-bf489c6f-cfb0-437a-a09d-4050856137a6.png)

**Comment by ZCMax** [comment]
please switch to dev-1.x branch, we will fix the problem soon
